### DISCUSSION TITLE
What should I do?

### DISCUSSION THREAD

**Discussion by Wh1tezhihei** [LSPosed/MagiskOnWSALocal#165] [Q&A] [answered]
![Ubuntu_ZHIHEI-2022-10-03-12-23-59](https://user-images.githubusercontent.com/114866978/193501510-3ef3dfa3-d869-4859-9333-e42435bebfa7.png)
Ubuntu 22.04.1 LTS

**Comment by Howard20181** [comment]
`sudo apt update && sudo apt dist-upgrade`

**Reply by Wh1tezhihei** [reply]
Thank you for your help, the above prompt is that my software version is too high, your instructions can not make my software downgrade, finally I delete the above software and run run.sh, the problem is solved.

**Comment by Wh1tezhihei** [comment, marked as answer]
use  **sudo apt-get --purge remove<>**  to remove the above software，then run the **run.sh**.
